test/Dialect/FIRRTL/layers-errors.mlir

```diff
@@ -169,3 +169,106 @@ firrtl.circuit "Top" {
     }
   }
 }
+
+// -----
+
+// XMRRefOp's result type must have an appropriate color.
+firrtl.circuit "Top" {
+  firrtl.layer @A bind {}
+  hw.hierpath @xmr [@Top::@wire]
+  firrtl.module @Top(out %o : !firrtl.probe<uint<1>>) {
+    firrtl.layerblock @A {
+      %wire = firrtl.wire sym @wire : !firrtl.uint<1>
+    }
+    %0 = firrtl.xmr.ref @xmr : !firrtl.probe<uint<1>>
+  }
+}
```

Comment:
If this is checking for something that _should_ be legal, then it should be put in a test that isn't a `-error` test.

Reply:
Sorry--this wasn't meant to be checked in. Removed!